src/status_im/anon_metrics/transformers.cljs

```diff
@@ -1,14 +1,24 @@
 (ns status-im.anon-metrics.transformers)
 
 (defn navigate-to-txf [context]
-  {:view-id (second context)
-   :params (-> context
-               (nth 2)
-               (select-keys [:screen]))})
+  (let [event (-> context :coeffects :event)]
+    {:view_id (second event)
+     :params  {:screen (get-in event [1 :screen] "")}}))
 
-(def transformations
-  {:navigate-to navigate-to-txf})
+(defn screens-on-will-focus-txf [context]
+  (let [view-id (-> context :coeffects :event second)]
+    ;; we need this check because in some cases, :screens/on-will-focus is dispatched after navigate to
+    ;; this check ensures that only the events that are not handled via navigate to are captured
+    ;; hence avoiding double entry
+    (when (some #(= view-id %) [:chat :empty-tab :wallet :status :my-profile])
```

Comment:
`(contains? #{:chat :empty-tab :wallet :status :my-profile} view-id)` requires less parentheses, not a big deal though

Reply:
`(#{:chat :empty-tab :wallet :status :my-profile} view-id)` even better